lib/dialects/sqlite3/schema/internal/sqlite-ddl-operations.js

```diff
@@ -37,12 +31,8 @@ function dropOriginal(tableName) {
   return `DROP TABLE "${tableName}"`;
 }
 
-function dropTempTable(alteredName) {
-  return `DROP TABLE "${alteredName}"`;
-}
-
 function renameTable(tableName, alteredName) {
-  return `ALTER TABLE "${tableName}" RENAME TO "${alteredName}"`;
+  return `ALTER TABLE "${alteredName}" RENAME TO "${tableName}"`;
```

Comment:
why was this swapped?

Reply:
Because the temporary altered table has to be renamed to the original table name. The function was only used in `renameTable()` in `ddl.js`, which in turn was unused.

Reply:
I would then change the passed params, because original naming implies "[original]Name" -> "alteredName", and now it is confusing.